webpack.config.js

```diff
@@ -125,10 +128,10 @@ if ( TARGET_ENV === 'production' ) {
         },
       ]),
 
-      new webpack.optimize.OccurenceOrderPlugin(),
+      new webpack.optimize.OccurrenceOrderPlugin(),
```

Comment:
This is [on by default](https://webpack.js.org/guides/migrating/#occurrenceorderplugin-is-now-on-by-default) and can therefore be removed